resources/views/maps/groups/create.blade.php

```diff
@@ -27,20 +27,47 @@
         @endif
         <div class="panel-body">
```

Comment:
Can't use `panel-body` in a modal, needs to be `modal-body` when it's an ajax request, and panel-body when not. 
![image](https://user-images.githubusercontent.com/3263440/196011085-3c9f35d7-8a93-4e15-8aff-7a43b006cfa4.png)